# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/webidl/MediaStreamTrack.webidl
# Commit: dc3e09fb750b
# Full Hash: dc3e09fb750bd5eea290e6acb81f31c4e20d74fd
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2019-07-10 15:45:05
# Regressor Bug: 1560907
# File Overlap Count: 1
# Description:
#   Bug 1560907 - Remove default value for mediaSource constraint. r=jib,smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D36087
# ==============================================================================

diff -r be4f68034abd -r dc3e09fb750b dom/webidl/MediaStreamTrack.webidl
--- a/dom/webidl/MediaStreamTrack.webidl	Wed Jul 10 09:43:53 2019 +0000
+++ b/dom/webidl/MediaStreamTrack.webidl	Wed Jul 10 09:45:26 2019 +0000
@@ -46,7 +46,7 @@
     ConstrainLong height;
     ConstrainDouble frameRate;
     ConstrainDOMString facingMode;
-    DOMString mediaSource = "camera";
+    DOMString mediaSource;
     long long browserWindow;
     boolean scrollWithPage;
     ConstrainDOMString deviceId;
